ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh

```diff
@@ -196,7 +196,7 @@ do
 
     extract_oc
 
-    if check_ota_case_enabled "OCP-30832"; then
+    if check_ota_case_enabled "OCP-30832" "OCP-69968"; then
```

Comment:
This if block is for `OCP-30832` case, mirror payload to a custom repo, and create signature together when mirroring, `OCP-69968` also need the steps?

Reply:
yes, ocp-69968 needs a mirror with signature together.